packages/crayons-core/src/components/file-uploader-2/file-uploader-2.tsx

```diff
@@ -0,0 +1,891 @@
+    return (
+      <div
+        class={{
+          'file-uploader__body__dropzone': true,
+          'file-uploader__body__dropzone--disabled':
+            this.isBatchUploadInProgress,
+          'file-uploader__body__dropzone--error': !!this.errors.length,
+        }}
+        key='dropzone'
+        tabIndex={0}
+        onDrop={(event) =>
+          !this.isBatchUploadInProgress && this.dropHandler(event)
+        }
+        onDragOver={(event) => event.preventDefault()}
+        onClick={() =>
+          !this.isBatchUploadInProgress && this.fileInputElement.click()
+        }
+        onKeyUp={(event) => {
+          if (event.key === 'Enter' || event.key === 'Space') {
+            !this.isBatchUploadInProgress && this.fileInputElement.click();
+          }
+        }}
+        ref={(el) => (this.dropzoneContainer = el)}
+        role='button'
+      >
+        <div class='file-uploader__body__dropzone__center'>
+          <div class='file-uploader__body__dropzone__center__clickable'>
+            <div class='file-uploader__body__dropzone__center__clickable__icon'>
+              {!this.errors.length ? (
+                <div
+                  innerHTML={
+                    new DOMParser().parseFromString(fileDragSVG, 'text/html')
+                      .body.innerHTML
+                  }
+                ></div>
+              ) : (
+                <div
+                  innerHTML={
+                    new DOMParser().parseFromString(fileErrorSVG, 'text/html')
+                      .body.innerHTML
+                  }
+                ></div>
+              )}
+            </div>
+            {!this.errors.length ? (
+              <div
+                class='file-uploader__body__dropzone__center__clickable__text'
+                innerHTML={
+                  this.text || TranslationController.t('fileUploader2.text')
+                }
+              ></div>
+            ) : (
+              <div class='file-uploader__body__dropzone__center__clickable__error'>
+                {this.errors[0]}.{' '}
+                <span class='highlight'>
+                  {TranslationController.t('fileUploader2.retry')}
+                </span>
+              </div>
+            )}
+            {this.description && (
+              <div class='file-uploader__body__dropzone__center__clickable__description'>
+                <span>{this.description}</span>
+              </div>
+            )}
+          </div>
+        </div>
+      </div>
+    );
+  }
+
+  /**
+   * renderFiles
+   * @returns {JSX.Element}
+   */
+  renderFiles() {
+    return this.files.length ? (
+      <div class='file-uploader__body__files' key='files'>
+        <div
+          class={{
+            'file-uploader__body__files__restrict':
+              this.restrictAttachmentWidth,
+          }}
+        >
+          <div class='file-uploader__body__files__center'>
+            {this.files.map((file) => {
+              return (
+                <fw-file-2
```

Comment:
should we name this component also with `-2` ?. I understand the reasoning behind doing it for `file-uploader-2`.

Reply:
Yes, this has the same reasoning as the primary component. If we append name with features, it would be hard to understand what to use.